### DISCUSSION TITLE
Welcome to markmap Discussions!

### DISCUSSION THREAD

**Discussion by gera2ld** [markmap/markmap#65] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wyaoting** [comment]
How to display images in markmap ？

**Reply by IGitYou** [reply]
the examples in the demo do not have one and I am guessing it does not support them? New here... I am commenting not answering.

**Reply by wyaoting** [reply]
Thank you very much for your message.

**Reply by eyssette** [reply]
You can use the markdown syntax for images : `![](URL)`. But on the [REPL](https://markmap.js.org/repl), the image will be displayed only if you also have some text in the same node.
Btw, I've created an online mindmap editor with markmap that can display images : https://mymarkmap.netlify.app/

**Reply by wyaoting** [reply]
Thank you very much for your answer
